app/controllers/mixins/ems_common_angular.rb

```diff
@@ -465,12 +467,14 @@ def build_credentials(ems, mode)
                          :save          => (mode != :validate)}
         session[:oauth_response] = nil
       end
-      if ems.kind_of?(ManageIQ::Providers::ContainerManager) &&
-         ems.supports_authentication?(:bearer) && !params[:default_password].blank?
-        creds[:hawkular] = {:auth_key => params[:default_password], :userid => "_"}
-        creds[:bearer] = {:auth_key => params[:default_password]}
+      if ems.kind_of?(ManageIQ::Providers::ContainerManager)
```

Comment:
@yaacov any way to remove this nasty check? Anything else that we can test for other than `kind`?